lib/eslint/eslint-helpers.js

```diff
@@ -47,6 +58,30 @@ class NoFilesFoundError extends Error {
     }
 }
 
+/**
+ * The error type when a search fails to match multiple patterns.
+ */
+class UnmatchedSearchPatternsErrror extends Error {
```

Comment:
```suggestion
class UnmatchedSearchPatternsError extends Error {
```

Typo (three r in Errror).